api/files.js

```diff
@@ -1,18 +1,24 @@
 /* jshint esversion: 6 */
 
-var promise = require('bluebird');
-var fs = promise.promisifyAll(require('fs'), {
+var _Promise = require('bluebird');
```

Comment:
Yes. It is technically incorrect to use `Promise` here because that overwrites the global `Promise` defined in ES6. Ideally we'd change all the `Promise`s in the codebase that are not ES6 promises to `_Promise`, but I'm not going to do that in this feature.